resilience4j-documentation/src/docs/asciidoc/addon_guides/springboot.adoc

```diff
@@ -21,6 +21,30 @@ dependencies {
 Spring Boot Actuator health information can be used to check the status of your running application.
 It is often used by monitoring software to alert someone if a production system has serious issues.
 
+==== Retry
+When you enable retry ,Metrics are automatically published on the Metrics endpoint.
```

Comment:
Could you please fix the space infront of the comma.

Reply:
Done :)